CHANGELOG.rst

```diff
@@ -12,12 +12,15 @@ Added
 - Added ``uni_a`` and ``uni_z`` to ``attributes_requiring_redeploy``
 - Added ``metadata`` to EVC schema
 - Allow the creation of ``any`` and ``untagged`` EVC.
+- Added api request ``POST /v2/evc/metadata`` to add metadata to EVCs
+- Added api request ``DELETE /v2/evc/metadata/<key>`` to delete metadata from EVCs
 
 Changed
 =======
 - Hid ui primary and secondary constraints on ``k-toolbar`` in the meantime
 - Moved request circuit ``k-button`` out of k-accordion-item since it's mandatory
 - The traces being check rely on ``type``: ``last`` to be considered valid.
+- Augmented ``GET /v2/evc/`` to accept parameters ``metadata.key=item``
```

Comment:
Can you also document this query arg in the openapi? Looks like currently we only support int, str and bool, so let's make that clear in the spec too. 

```
          in: query
          schema:
            type: ...
```